src/scripts/main.js

```diff
@@ -21,7 +21,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    const item = document.createElement('li');
+
+    item.textContent = key;
+
+    list.appendChild(item);
+
+    if (Object.keys(data).length !== 0) {
```

Comment:
The condition `if (Object.keys(data).length !== 0)` is checking if the current object has any keys. However, this condition will always be true for the current implementation since you are iterating over the keys of `data`. Instead, you should check if `data[key]` has keys, i.e., `if (Object.keys(data[key]).length !== 0)`, to decide whether to recurse.